engine/pinball_game/object_collision/blue_stage_resolve_collision.asm

```diff
@@ -5929,7 +5929,7 @@ GoToBonusStageTextIds_BlueField:
 	db STAGE_DIGLETT_BONUS
 	db STAGE_SEEL_BONUS
 
-Func_1e8f6: ; 0x1e8f6
+QueueGraphicsDataToLoad: ; 0x1e8f6

```

Comment:
Func_1e8f6 was the function I was talking about when we were discussing a name, I thought that we were deciding a name for that function. Oops.